packages/deribit/lib/index.ts

```diff
@@ -0,0 +1,52 @@
+import { OptionType } from '@node-dlc/core';
+
+export const getStrDate = (timestamp: number): string => {
+  const date = new Date(timestamp * 1000);
+
+  const day = date.getUTCDate();
+  const month = date
+    .toLocaleString('default', { month: 'short', timeZone: 'UTC' })
+    .toUpperCase()
+    .replace('.', '');
+  const year = date.toLocaleDateString('pl', {
+    year: '2-digit',
+    timeZone: 'UTC',
+  });
+
+  return `${day}${month}${year}`;
+};
+
+type OptionInstrument = {
+  expiry: number;
+  strikePrice: number;
+  type: OptionType;
+};
+
+export const composeInstrumentName = ({
+  expiry,
+  type,
+  strikePrice,
+}: OptionInstrument): string => {
+  const strDate = getStrDate(expiry);
+  const typeSymbol = type === 'call' ? 'C' : 'P';
+  return `BTC-${strDate}-${strikePrice}-${typeSymbol}`;
+};
+
+const EXPIRY_REGEX = /(\d{1,2})([A-Z]+)(\d{1,2})/;
+export const parseInstrumentName = (
+  instrumentName: string,
+): OptionInstrument => {
```

Comment:
what should `parseInstrumentName` return if non-option instrumentName passed in (i.e. `BTC-PERPETUAL`)? might be good to return null and wrap with a `try` `.catch()`